packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -102,45 +112,54 @@ export default function CardSection({
               isSection
             />
           </button>
-          {actions && <Actions actions={actions} />}
+          {actions && (
+            <Stack inline grow={false} justify="end">
+              {actions}
+            </Stack>
+          )}
         </div>
       )}
 
       {(title != null || header != null) && !expandable && (
-        <div className="p-400 lm:p-600 w-full">
-          <Header
-            title={title}
-            titleAs={titleAs}
-            description={description}
-            expandable={expandable}
-            header={header}
-            expanded={opened}
-            actions={actions}
-            isSection
-          />
-          {actions && <Actions actions={actions} />}
+        <div className="p-400 lm:p-600 ">
+          <InteractiveWrapper onClick={onClick}>
+            <Header
+              title={title}
+              titleAs={titleAs}
+              description={description}
+              expandable={expandable}
+              header={header}
+              expanded={opened}
+              actions={actions}
+              isSection
+            />
+          </InteractiveWrapper>
         </div>
       )}
 
       {children && expandable && (
         <Expandable expanded={opened} slideID={slideID} labelID={slideID}>
           <div className="font-base text-normal text-primary-foreground px-400 lm:px-600 w-full leading-normal">
-            <div className="py-400 lm:py-600 border-elevation-flat-border-color border-t">
+            <InteractiveWrapper
+              onClick={opened ? onClick : undefined}
```

Comment:
`onClick` is available only when `opened=true`, if not, the component would be focusable twice (in case it's not collapsable and has children).